### ISSUE TITLE
Is it possible to pass linux environment variables?

### ISSUE THREAD

**Issue by com2ghz** [fastlane/fastlane#8490]
I am using Fastlane to release 400 Android apps. It works like a charm, but the only problem is that I need to provide the location of the json_key_file.

How can I configure my json_key_file in one place which applies to all 400 apps? I tried to use .env file inside my fastlane directory in my home dir but this didn' t work.

Is there a way to do this?

**Comment by mfurtak** [comment]
Hi @com2ghz :wave: 

Have you seen the info at https://docs.fastlane.tools/advanced/#environment-variables ?

If that's not working for you, can you be more specific about the problems you're seeing? Thanks! 👍

**Comment by com2ghz** [comment]
Hey thank you for the replies. I found out what I did wrong. I forgot to export the variable. So I only had GOOGLE_JSON_KEY="/home/...." but forgot to add export in front of it.

Now it works like a charm.

I think the documentation should be more clear for Android. My colleague is developing the IOS variant of the apps but he said that he could put the '.env' file inside his .fastlane directiry in his home dir.
